static/css/legacy.less

```diff
@@ -185,6 +185,9 @@ table {
       width: 33%;
     }
   }
+  a.blocked-ip {
```

Comment:
Rather than add this new rule, make use of the existing css rule ".invalid", ".red", .attn or ".neg" which doesn't exactly the same thing!

Reply:
Okay i will modify this rule